data/domains/movies.yaml

```diff
@@ -1,6 +1,6 @@
 slot_names:
```

Comment:
We should assume that all items have a reserved ID slot, and document it in domain.py and also in the docs.

Reply:
`domain.py` is part of DK. I created an (issue)[https://github.com/iai-group/DialogueKit/issues/208] to add the `ID` field and update documentation.

Reply:
Done.